lib/IRGen/GenDistributed.cpp

```diff
@@ -693,7 +693,7 @@ void DistributedAccessor::emit() {
   }
 
   // Add all of the substitutions to the explosion
-  if (auto *genericEnvironment = Target->getGenericEnvironment()) {
+  if (Target->isGeneric()) {
```

Comment:
Heh, there is no `isGeneric()` on 5.10 but there is on main -- I'll revert this single line; functionally it is the same.

Reply:
Sounds good to me!